app/views/posts/index.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<h1>SECRET POSTS</h1>
+
+<table>
+  <tr>
+    <th>Usuario</th>
+    <th>Title</th>
+    <th>Text</th>
+  </tr>
+
+  <% @post.each do |article| %>
+    <tr>
+      <% if logged_in? %>
```

Comment:
- [ ] It's against recommended to put any logic in the views. Rails has built-in mechanism to deal with view-related logic which is View Helpers (app/helpers/*_helper.rb). So the common convention is to move any logic in the views to the helpers. Please take a look at [this](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for more information.